app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -124,6 +132,8 @@ def invalid_otp_error(type)
     case type
     when 'otp'
       t('two_factor_authentication.invalid_otp')
+    when 'totp'
+      t('two_factor_authentication.invalid_otp')
```

Comment:
I guess it's worth calling out for others: the code previously passed in "otp" for both phone-based OTP and TOTP, but we wanted to distinguish between them as distinct events.

This way we pass in extra detail, but don't change the user experience.